back/config/environments/development.rb

```diff
@@ -43,8 +43,8 @@
 
   config.action_mailer.perform_caching = false
 
-  # Previewing mails from the email engine.
-  config.action_mailer.preview_path = "#{Rails.root}/engines/free/email_campaigns/spec/mailers/previews"
+  # Previewing mails from all engines.
+  config.action_mailer.preview_path = "#{Rails.root}/engines/**/**/spec/mailers/previews"
```

Comment:
Nice again 👍

Reply:
Nothing too important, but a bit surprised by the double `**/**`. I would have expected `**` to be sufficient :-)